### DISCUSSION TITLE
how to load my own labels of behavior?

### DISCUSSION THREAD

**Discussion by haolapig** [healthonrails/annolid#56] [General]
Excellent tool!
I have a video of two mice interacting with each other, and I have labeled the interaction behaviors frame by frame using another tool. The behavior labels are based on two mice and are marked with start and stop times, similar to the process in Annolid. Do you have any suggestions on how to load my behavior label data into Annolid and train my own model?

**Comment by healthonrails** [comment]
Thank you for your interest in Annolid! It’s great to hear that you’ve already labeled your behavior data frame by frame for your video. We are currently working on features that utilize spatio-temporal masks or masklets for behavior analysis.

Would you be interested in collaborating by sharing your videos and labeled files? This would allow us to explore the best approach for integrating your data and determining the next steps.

**Reply by haolapig** [reply]
Yes, we are definitely interested in collaborating and sharing the data and labels we have. Please provide the official contact and outline the process to move forward with this.

**Comment by healthonrails** [comment]
Sounds great! Could you please share your data and labels via Google Drive or another downloadable link? We can also schedule a meeting if you’d like to discuss the details.

**Comment by haolapig** [comment]
Hi, thank you for the updates.

I would like to introduce myself – I am a research engineer at the
University of Iowa. I was hoping to obtain your official contact
information, preferably an institutional email address rather than a GitHub
account, which you can send to ***@***.***

Once I have your details, I can arrange a meeting with the data owners, Dr.
Shane Heiney and Dr. Sarah Ferri, who have already agreed to share the
datasets. I will share the data through the University’s OneDrive shared
folder.

Currently, we have the following datasets available:

   - 12 control videos (~10 minutes each, all labeled)
   - 7 action videos (~10 minutes each, all labeled)

We also have additional data available if needed.

Best regards,
Zhi Chen

**Comment by healthonrails** [comment]
You can find our contact information in this paper: https://arxiv.org/pdf/2403.18690.
